boms/cloud-lts-bom/pom.xml

```diff
@@ -47,12 +47,9 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    
-    <!-- interrim placeholder versions -->
-    <!-- TODO: For each library, update the versions to the LTS version -->
+
+    <!-- finalized versions -->
```

Comment:
I think you can drop this comment completely.

Reply:
Removed this line.